scripts/monitor/handlers/index.js

```diff
@@ -1,10 +1,109 @@
 // Check the default file search progression on './HandlerIndex#get'
 
-export function transform_Marketplace_Publish(event) {
+import { eth, txUtils, contracts } from 'decentraland-commons'
+import { decodeAssetId, debounceById } from './utils'
+import { Parcel } from '../../../src/Parcel'
+import { Publication } from '../../../src/Publication'
+
+export async function transform_Marketplace(event) {
+  if (event.removed) return
+
+  const { transactionHash } = event
+  const { assetId } = event.args
+  const id = await decodeAssetId(assetId)
+
+  switch (event.event) {
+    case 'AuctionCreated': {
+      const { priceInWei, expiresAt } = event.args
+      const [x, y] = Parcel.splitId(id)
+
+      console.log(
+        `[Marketplace-AuctionCreated] Creating publication for ${x} ${y}`
+      )
+
+      await Publication.insert({
+        tx_hash: transactionHash,
+        tx_status: txUtils.TRANSACTION_STATUS.confirmed,
+        status: Publication.STATUS.open,
+        owner: null, // TODO: Add owner
+        buyer: null,
+        price: eth.utils.fromWei(priceInWei.toNumber()),
+        expires_at: new Date(expiresAt),
+        x,
+        y
+      })
+      break
+    }
+    case 'AuctionSuccessfull': {
+      const { totalPrice, winner } = event.args
+
+      console.log(
+        `[Marketplace-AuctionSuccessfull] Publication ${id} sold to ${winner}`
+      )
+
+      await Publication.update(
+        {
+          status: Publication.STATUS.sold,
+          buyer: winner,
+          price: eth.utils.fromWei(totalPrice.toNumber())
+        },
+        { id }
+      )
+      await Parcel.update({ owner: winner }, { id })
+      break
+    }
+    case 'AuctionCancelled': {
+      console.log(`[Marketplace-AuctionCancelled] Publication ${id} cancelled`)
+
+      await Publication.update({ status: Publication.STATUS.cancelled }, { id })
+      break
+    }
+    default:
+      break
+  }
+
   return event
 }
 
-export function transform_LANDRegistry_Transfer(event) {
+export async function transform_LANDRegistry(event) {
+  if (event.removed) return
+
+  const { assetId } = event.args
+  const id = await decodeAssetId(assetId)
+
+  switch (event.event) {
+    case 'Update': {
```

Comment:
Shouldn't this be `'Transfer'` and the case below `'Update'` instead?

Reply:
Whoops good catch!